integrationTests/p2p/antiflood/messageProcessor.go

```diff
@@ -0,0 +1,79 @@
+package antiflood
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+type messageProcessor struct {
+	numMessagesProcessed  uint32
+	sizeMessagesProcessed uint64
+
+	numMessagesReceived  uint32
+	sizeMessagesReceived uint64
+
+	mutMessages    sync.Mutex
+	messages       map[p2p.PeerID][]p2p.MessageP2P
+	FloodPreventer process.FloodPreventer
+}
+
+func newMessageProcessor() *messageProcessor {
+	return &messageProcessor{
+		messages: make(map[p2p.PeerID][]p2p.MessageP2P),
+	}
+}
+
+func (mp *messageProcessor) ProcessReceivedMessage(message p2p.MessageP2P, fromConnectedPeer p2p.PeerID, _ func(buffToSend []byte)) error {
+	atomic.AddUint32(&mp.numMessagesReceived, 1)
+	atomic.AddUint64(&mp.sizeMessagesReceived, uint64(len(message.Data())))
+
+	if mp.FloodPreventer != nil {
+		//protect from directly connected peer
+		ok := mp.FloodPreventer.TryIncrement(string(fromConnectedPeer), uint64(len(message.Data())))
```

Comment:
Increment as name for the function is enough. as it returns error if he cannot.

Reply:
Increment shall be